modules/phase_field/src/ics/PolycrystalColoringIC.C

```diff
@@ -36,8 +36,15 @@ PolycrystalColoringIC::PolycrystalColoringIC(const InputParameters & parameters)
 Real
 PolycrystalColoringIC::value(const Point & p)
 {
+  _poly_ic_uo.getGrainsBasedOnPoint(p, _grains);
```

Comment:
Here's the general Polycrystal IC class that gets what it needs from the UserObject. Here you can see me grabbing a vector of `_grains` at a point.